modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,481 @@
+If you reuse the `BareMetalHost` object definition from an existing control plane host, do not leave the `externallyProvisioned` field set to `true`.
+
+Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by the {product-title} installation program.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have taken an etcd backup.
++
+[IMPORTANT]
+====
+Take an etcd backup before performing this procedure so that you can restore your cluster if you encounter any issues. For more information about taking an etcd backup, see the _Additional resources_ section.
+====
+
+.Procedure
+
+. Ensure that the Bare Metal Operator is available:
++
+[source,terminal]
+----
+$ oc get clusteroperator baremetal
+----
++
+.Example output
+[source,terminal]
+----
+NAME        VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+baremetal   4.10.12   True        False         False      3d15h
+----
+
+. Remove the old `BareMetalHost` and `Machine` objects:
++
+[source,terminal]
+----
+$ oc delete bmh -n openshift-machine-api vmaster-0
+$ oc delete machine -n openshift-machine-api kni1-master-0
+----
++
+After you remove the `BareMetalHost` and `Machine` objects, then the `Machine` controller automatically deletes the `Node` object.
+
+. Create the new `BareMetalHost` object and the secret to store the BMC credentials:
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kni1-master-0-bmc-secret
+  namespace: openshift-machine-api
+data:
+  password: <username>
+  username: <password>
+type: Opaque
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: kni1-master-0
+  namespace: openshift-machine-api
+spec:
+  automatedCleaningMode: disabled
+  bmc:
+    address: redfish-virtualmedia+http://192.168.124.113:8000/redfish/v1/Systems/f87eaf82-b32d-4291-ace6-b28677964e78
+    credentialsName: kni1-master-0-bmc-secret
+  bootMACAddress: aa:aa:aa:aa:ab:03
+  bootMode: UEFI
+  externallyProvisioned: false
+  hardwareProfile: unknown
+  online: true
+EOF
+----
++
+After the inspection is complete, the `BareMetalHost` object is created and available to be provisioned.
+
+. View available `BareMetalHost` objects:
++[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
+NAME           STATE                    CONSUMER                   ONLINE   ERROR   AGE
+kni1-master-0  available                ocp-hkw9p-master-0         true             1h10m
+kni1-master-1  externally provisioned   ocp-hkw9p-master-1         true             4h53m
+kni1-master-2  externally provisioned   ocp-hkw9p-master-2         true             4h53m
+kni1-worker-0  provisioned              ocp-hkw9p-worker-0-ktmmx   true             4h53m
+kni1-worker-1  provisioned              ocp-hkw9p-worker-0-l2zmb   true             4h53m
```

Comment:
on this stage the new BMH have no consumer yet. 
$ oc get bmh -n openshift-machine-api
NAME                 STATE                    CONSUMER                        ONLINE   ERROR   AGE
openshift-master-0   externally provisioned   ocp-edge-4zg89-master-0         true             4h6m
openshift-master-1   externally provisioned   ocp-edge-4zg89-master-1         true             4h6m
openshift-master-2   available                                               true             35m14s
openshift-worker-0   provisioned              ocp-edge-4zg89-worker-0-8v2jj   true             4h6m
openshift-worker-1   provisioned              ocp-edge-4zg89-worker-0-jb5z9   true             4h6m

Reply:
Done